backend/pennclubs/settings/ci.py

```diff
@@ -11,7 +11,7 @@
 TEST_OUTPUT_DIR = "test-results"
 
 # Use dummy cache for testing
-CACHES = {"default": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"}}
+CACHES = {"default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"}}
```

Comment:
Caching was effectively turned off in CI/CD tests -- not sure what the original intention was, but I'm reverting it back. We interact with the cache manually in a lot of our views, so feels reasonable that the unit tests also check its correctness.